hack/legacy/controller_patch.json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
we typically keep the patches under `build/yamls/patches`; I think legacy can be moved there